lib/Conversion/StableHLOToTTIR/StableHLOToTTIRPatterns.cpp

```diff
@@ -338,10 +338,48 @@ class StableHLOToTTIRConstantOpConversionPattern
     return success();
   }
 
+  // TTNN does not support boolean data type; so booleans are converted to
+  // bfloat16 data type.
+  // No modification required for integer and float types.
+  mlir::ElementsAttr getValueAttrForTensor(mlir::ElementsAttr valueAttr) const {
```

Comment:
Is this the right place to handle int64 conversation as well?

Reply:
Similar to boolean types, we need to handle 64-bit types in two places:

1. For constants with 64-bit types, it should be handled similar to boolean types as this PR.

2. For function signatures with 64-bit types, we should add it to type converter. e.g. 
```
  if (type.getElementTypeBitWidth() == 64) {
      elementType = BFloat16Type::get(elementType.getContext());
      return RankedTensorType::get(shape, elementType);
  }
```

Reply:
I'll submit another PR to handle 64-bit integers.

Reply:
64-bit integer conversion is added in this PR.